docs/modules/ROOT/pages/reference/_nav.adoc

```diff
@@ -0,0 +1,7 @@
+* xref:reference/index.adoc[Reference]
```

Comment:
Do you want to move the glossary into the reference section as well?

Reply:
I like it, but would consider this as a follow-up action. I'm working on a broader docs re-arch/reorg proposal.